### ISSUE TITLE
Bug: `hook-use-state` false positive

### ISSUE THREAD

**Issue by zaicevas** [jsx-eslint/eslint-plugin-react#3324]
Version: 7.30.1

This code snippet should not produce an error. However, it does.

```tsx
const [uuid, setUuId] = useState('')
```

**Comment by zaicevas** [comment]
Whoops, my bad. Haven't noticed an uppercase `I` 😶
